app/models/donation.rb

```diff
@@ -48,4 +50,8 @@ class Donation < ActiveRecord::Base
 	belongs_to :event
 
 	scope :anonymous, -> {where(anonymous: true)}
+
+	private def set_anonymous
```

Comment:
Minor nitpick but best practices for ruby generally have the private on it's own line. That, of course, means everything in the file after it is private.